jit/v2/JIT_v2.0.0.yaml

```diff
@@ -270,6 +493,102 @@ paths:
           headers:
             API-Version:
               $ref: '#/components/headers/API-Version'
+        '400':
+          description: |
+            In case cancelling a `Port Call Service` does not schema validate a `400` (Bad Request) is returned
```

Comment:
In case cancelling a 'Port Call Service' fails schema validation a 400 (Bad Request) is returned